manifests/init.pp

```diff
@@ -398,6 +393,13 @@
     }
   }
 
+  if $pulp or $pulp_master {
+    class { 'crane':
```

Comment:
I suspected it might need a dependence on something, but wasn't sure what.  thanks!